packages/client/utils/AtlassianManager.ts

```diff
@@ -42,6 +42,9 @@ export interface JiraProject {
   }
   simplified: boolean
   style: string
+  cloudId: string
```

Comment:
-1 these values aren't coming from the Jira API, so we can't list them here. let's handle adding those in the calling function (maybe fetchAtlassianProjects?) we can augment the return value in the dataloader